### PR TITLE
[doc] Add an operation page

### PR DESCRIPTION
**Pull request by qiao-bo**
Move the supported operations content out of the type system to a new page. Also add supported matrices operations from old docs. 

related issue #3318

### CODE REVIEW

File: docs/lang/articles/basic/operator.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+| `a > b`            | if `a` is strictly greater than `b`, then True, else False    |
+| `a < b`            | if `a` is strictly less than `b`, then True, else False       |
+| `a >= b`           | if `a` is greater than or equal to `b`, then True, else False |
+| `a <= b`           | if `a` is less than or equal to `b`, then True, else False    |
+| `not a`            | if `a` is False, then True, else False                        |
+| `a or b`           | if `a` is False, then `b`, else `a`                           |
+| `a and b`          | if `a` is False, then `a`, else `b`                           |
+| `a if cond else b` | if `cond` is True, then `a`, else `b`                         |
+
+### Bitwise operators
```
Comment (lin-hitonami) [inline docs/lang/articles/basic/operator.md:75]: `<<` and `>>` are not listed in the original document. Should we add them too?

File: docs/lang/articles/basic/operator.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+ti.sqrt(x)
+ti.rsqrt(x)  # A fast version for `1 / ti.sqrt(x)`.
+ti.exp(x)
+ti.log(x)
+ti.round(x)
+ti.floor(x)
+ti.ceil(x)
+```
+
+### Builtin-alike functions
```
Comment (lin-hitonami) [inline docs/lang/articles/basic/operator.md:108]: maybe add `sum`?

File: docs/lang/articles/basic/operator.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
+---
+sidebar_position: 4
+---
+
+# Operators
+Here we present the supported operators in Taichi for both primitive types and
+compound types such as matrices.
+
+## Supported operators for primitive types
+### Arithmetic operators
```
Comment (lin-hitonami) [inline docs/lang/articles/basic/operator.md:10]: We also support `@`.

**Review by Vissidarte-Herman — APPROVED** [review]

**Review by lin-hitonami — APPROVED** [review]
LGTM!

**Pull request merged**